src/PrestaShopBundle/Controller/Admin/Configure/ShopParameters/CustomerGroupsController.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace PrestaShopBundle\Controller\Admin\Configure\ShopParameters;
+
+use PrestaShop\PrestaShop\Core\Search\Filters\CustomerGroupsFilters;
+use PrestaShop\PrestaShop\Core\Search\Filters\TitleFilters;
+use PrestaShopBundle\Controller\Admin\FrameworkBundleAdminController;
+use PrestaShopBundle\Security\Annotation\AdminSecurity;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
```

Comment:
You must use php strict typing for new classes

Reply:
and license header